SparkPost/2web2ui
# AC-1089: Add link to API keys page from checklist

### What Changed
 - Link to `/account/api-keys` from onboarding checklist

### How To Test
 - Use feature flag `messaging_onboarding`
 - Go to `Messaging` > `Show Me Sparkpost` > `Let's Code` > `Generate API Key`
 - Generate API key with sending permission
 - Go back to dashboard and list should be updated with check
